docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
-```
+  let jobB = new Job("my-b-job", "debian", event);
+  jobB.primaryContainer.command = ["echo"];
+  jobB.primaryContainer.arguments = ["My B job!"];
 
-If we use `kubectl` to check out the log for our `test-app` container, we'll see something
-like this:
+  await Job.concurrent(
+    Job.sequence(job1, job2),
+    Job.sequence(jobA, jobB)
+  ).run();
+});
 
-```console
-$ kubectl logs test-app-1504064455281-master
-yarn install v0.27.5
-info No lockfile found.
-[1/4] Resolving packages...
-[2/4] Fetching packages...
-[3/4] Linking dependencies...
-[4/4] Building fresh packages...
-success Saved lockfile.
-Done in 0.14s.
-hello world
+events.process();
 ```
+[05-groups](https://github.com/brigadecore/brigade/tree/v2/examples/05-groups)
 
-That is the output of our three tasks.
+There are two things to notice in the example above:
 
-But what exactly is this build doing? Why are we doing a `cd /src/hello`? And where did
-that directory even come from?
+1. Both the `concurrent` and `sequence` methods exist on the `Job` object.
+2. The return type for both methods is a generic "runnable", i.e. an object
+  that `run()` can then be called on, just like a standalone job instance.
 
-Here's what is happening: Because our project has a Git repository associated with it,
-Brigade is automatically getting us a copy of that project and attaching that copy to each
-`Job` that we run.
+Here's a breakdown of each method:
 
-When a repository is checked out, it is stored by default in `/src`. So it is as if we
-started every job by doing a `git clone https://github.com/brigadecore/empty-testbed.git /src`.
-That means we can start our job knowing that we already have access to everything in our
-Git project.
+- `Job.sequence()` takes an array of runnables (e.g. a job or a group of
+  jobs) and runs them _in sequence_. A new runnable is started only when the
+  previous one completes. The sequence completes when the last runnable has
+  completed (or when any runnable fails).
+- `Job.concurrent()` takes an array of runnables and runs them all
+  _in parallel_. When run, all runnables are started simultaneously (subject to
+  scheduling constraints). The concurrent group completes when all Runnables
+  have completed.
 
-So when we `cd` into `/src/hello`, we're changing into the `hello/` directory in the Git
-project. And from there on, we are working with the code from the repository.
+As both of these methods return a runnable, they can be chained. In the example
+above, `job1` and `job2` run _in sequence_, as do `jobA` and `jobB`, but both
+sequences are run _in parallel_.
 
-Being able to associated a Git repository to a project is a convenient way to provide
-version-controlled data to our Brigade builds. And it makes Brigade a great tool for executing
-CI pipelines, deployments, packaging tasks, end-to-end tests, and other DevOps tasks.
+This is the way script writers can control the order in which groups of jobs
+are run.
```

Comment:
Worth mentioning a realistic usage of this?

A good example might be that if using Brigade to implement CI, you might wish to divide your checks into those that are resource intensive and those that are less so. Both groups can run the jobs within concurrently, but the groups themselves might be run in sequence, such that no resource intensive checks are executed unless/until all of the less resource intensive checks have passed.